Dockerfile

```diff
@@ -1,10 +1,23 @@
-FROM node:lts-slim
+FROM node:16.15.1-alpine as base
```

Comment:
Usage of the specific latest stable version. In order to avoid something breaking when a new version of node becomes the latest stable (e.g.: node 18)